libbeat/publisher/queue/diskqueue/queue.go

```diff
@@ -86,6 +91,18 @@ type diskQueue struct {
 	done chan struct{}
 }
 
+// channel request for metrics from an external client
+type metricsRequest struct {
+	response chan metricsRequestResponse
+}
+
+// metrics response from the disk queue
+type metricsRequestResponse struct {
+	sizeOnDisk    uint64
+	oldestEntryID segmentID
+	OccupiedRead  uint64
```

Comment:
Why is this the only public/exported struct member? I would expect either all or none of the fields to be exported.